src/unittests/f32_interval.spec.ts

```diff
@@ -913,7 +913,7 @@ g.test('inverseSqrtInterval')
       { input: 1, expected: [1, 1] },
       { input: 100, expected: [minusOneULP(hexToF32(0x3dcccccd)), hexToF32(0x3dcccccd)] },  // ~0.1
       { input: kValue.f32.positive.max, expected: [hexToF32(0x1f800000), plusNULP(hexToF32(0x1f800000), 2)] },  // ~5.421...e-20, i.e. 1/√max f32
-      { input: Number.POSITIVE_INFINITY, expected: [0, plusNULP(hexToF32(0x1f800000), 2)] },
+      { input: kValue.f32.infinity.positive, expected: arrayToInterval([Number.NEGATIVE_INFINITY, Number.POSITIVE_INFINITY]) },
```

Comment:
As far as I can tell, `arrayToInterval` now only exists to prevent the `[Number.NEGATIVE_INFINITY, Number.POSITIVE_INFINITY]` interval being expanded by error thresholds. If you were to adopt the `Any` suggestion, then the test body could check whether the interval is equal to the `Any` array (object equality would work), and if so, skip the error expansion logic.

If you did all this then `PointToIntervalCase.expected` could just become `expected: [number, number]`, and you could drop the `arrayToInterval` smell in the test cases.

WDYT?

Reply:
Nice. I like it.